.drone.yml

```diff
@@ -150,6 +150,7 @@ steps:
   image: buildkite/puppeteer
   commands:
   - yarn wait-on http://$HOST:$PORT
+  - yarn wait-on http://$HOST:$PORT/login
```

Comment:
Does this need to wait on both? Shouldn't this new line be enough?

Reply:
I agree, I guess we don't need this wait any longer?

Reply:
that's true, I don't think we need it anymore :)